mobile/src/app/components/protests/protests.component.scss

```diff
@@ -1,17 +0,0 @@
-.protests {
```

Comment:
Delete all the empty `scss` files, please. And remember the reference to them in their respective component.